CMakePresets.json

```diff
@@ -289,6 +344,16 @@
         "BUILD_SAMPLES": true
       }
     },
+    {
+      "name": "x86-msvc-static-debug-perftests",
+      "displayName": "x86 MSVC Debug static With Perf Tests and samples",
+      "inherits": "x86-msvc-static-debug-tests",
+      "cacheVariables": {
+        "BUILD_TRANSPORT_WINHTTP": true,
+        "BUILD_PERFORMANCE_TESTS": true,
+        "BUILD_SAMPLES": true
+      }
```

Comment:
It isn't clear what the difference is between `x86-msvc-static-debug-perftests` and `x86-msvc-static-debug-tests` from the name/description. It seems the only difference is that building on winhttp is true, in one.

Reply:
A lot of this was transcribed from our old cmakesettings.json files, which were sloppy. When I created the cmakepresets.json, I didn't want to break anyone's workflow so I copied a bunch of the stuff from cmakesettings.json.

THis is an example of that - -debug-tests is only supposed to enable build_testing and build_samples, the debug-perftests is supposed to enable perf tests on top of -debug-tests. 

I'll straighten this out because it's annoying.